arch/arm64/core/el3_service.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023 Chad Karaginides <quic_chadk@quicinc.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * This file implements the ARM64 EL3 service.
+ *
+ * Service available:
+ * Secure Monitor Call (SMC) handler.
+ *
+ * See https://developer.arm.com/docs/den0028/latest
+ */
+
+#include <zephyr/toolchain.h>
+#include <zephyr/arch/arm64/arm-smccc.h>
+
+typedef struct arm_smccc_res arm_smccc_cmd_rsp_t;
```

Comment:
why not reuse `typedef struct arm_smccc_res arm_smccc_res_t;` 
 at `include/zephyr/arch/arm64/arm-smccc.h L25`?

Reply:
The name of the struct is just "res" for response, not "cmd_res".  Response only was its original function.  I was just trying to make clear that it is also being used for command and response.  If that is not a big deal I can just use arm_smccc_res.  I will move to that.